src/components/flat-table/flat-table-row/flat-table-row.style.js

```diff
@@ -55,6 +60,16 @@ const StyledFlatTableRow = styled.tr`
   table-layout: fixed;
   width: auto;
 
+  ${StyledFlatTableCell} {
+    ${({ bgColor }) => bgColor && `background-color: ${bgColor};`}
```

Comment:
If we use the `toColor` util we could allow consumer's to use the palette as well as hex and css strings

https://github.com/Sage/carbon/blob/b1e510427d75bc191e35467f85ec93995f0ef8a3/src/style/utils/color.js